wsi/physicaldisplay.cpp

```diff
@@ -110,21 +110,16 @@ void PhysicalDisplay::DisConnect() {
 
   display_state_ &= ~kDisconnectionInProgress;
 
-  if (!(display_state_ & kConnected)) {
+  if (!display_queue_->Exit(power_mode_)) {
```

Comment:
We can just leave like now.

Reply:
But you only asked me to add the entry point in display_queue to do the disable. I hope my understanding is correct.

Reply:
Then I will add this cleanup in the disable path. Is that fine?